AmplifyPlugins/API/AWSAPICategoryPluginFunctionalTests/GraphQLSyncBased/GraphQLSyncBasedTests.swift

```diff
@@ -400,6 +400,113 @@ class GraphQLSyncBasedTests: XCTestCase {
         wait(for: [conflictUnhandledError], timeout: TestCommonConstants.networkTimeout)
     }
 
+    // Given: A newly created post
+    // When: Call delete mutation, twice
+    // Then: The first delete mutation is successful, and second returns conflict unhandled exception due to older version.
+    func testCreatePostThenDeleteTwiceConflictUnhandledException() {
```

Comment:
Is calling delete twice on an item a real use case for our software, or is this more or less a workaround because we don't have the mechanism to setup a scenario where we'd run into a conflict?

Reply:
this was one way i found to easily get the conflict unhandled exception for the integration tests, (test is removed though, we have another one that covers this scenario by double update). i would consider this scenario to be where both users (or same user on different device) has access to modify the model instance. does it at the same time, one wins and other fails. so even if we think of other scenarios, i think this one is still good to keep